workspace/assets/js.json

```diff
@@ -5,11 +5,15 @@
 		"utils/disable.js",
 		"utils/jquery.transitionend.js",
 		"utils/keys.js",
-		
 		"fx/tracking.js",
 		
 		"com/article-changer-v3.js",
 		"com/checkpoint-event.js",
+		"com/oembed.js",
+		"com/video.js",
+
+		"modules/oembed-yt.js",
+		"modules/oembed-vm.js",
```

Comment:
They should not be there by default

Reply:
@dominic-blain yes please remove and keep the only basic basic minimum.

Reply:
You are right video.js should not be here but youtube and vimeo should be there by default in the starter kit in 2019 (2020?). They work with module-oembed and every site should include one.

Reply:
@dominic-blain The thing is this: it is easier to see that the js file is missing, than see that it is unused...

So I would rather have them in the doc section of each xlt file.

Reply:
I understand that, and that's why I removed video.js since it could be unused but my point was that oembed must always be in every site. So basic things like this are the minimum.

But if you still want to remove it that will be my last plea against it.

Reply:
Done!